scripts/docker_cmake

```diff
@@ -7,6 +7,7 @@ ARGS=(
     -D BUILD_SHARED_LIBS=ON
     ### COMPILERS AND FLAGS ###
 #    -D CMAKE_CXX_FLAGS="-Wall -Wpedantic"
+    -D Trilinos_ENABLE_Fortran=ON
```

Comment:
Should we do this or add the extra flag to the `docker/jenkins/build.sh` script.  What do you all think?